src/app/app.module.ts

```diff
@@ -27,7 +27,7 @@ import { CovalentLoadingModule } from '@covalent/core/loading';
 import { CovalentMessageModule, CovalentStepsModule, CovalentDialogsModule } from '@covalent/core';
 import { CovalentHighlightModule } from '@covalent/highlight';
 import { CovalentBreadcrumbsModule } from '@covalent/core/breadcrumbs';
-import { CovalentHttpModule } from '@covalent/http';
+import { CovalentHttpModule } from '@covalent/http-deprec';
```

Comment:
I think we should not use the `http-deprec` version, b/c @angular/http will be removed soon from the angular framework and `http-deprec` will break when upgrading in future. If we can change the usage to the new https://teradata.github.io/covalent/#/components/http we can avoid having to make deprecated changes in the future.

Reply:
Spoke with Jenn, and we were thinking this can be done in a future PR outside of this scope when http-deprec gets removed.